cpp/src/strings/strings_column_factories.cu

```diff
@@ -114,9 +114,7 @@ std::unique_ptr<column> make_strings_column(size_type num_strings,
                                             std::unique_ptr<column> offsets_column,
                                             std::unique_ptr<column> chars_column,
                                             size_type null_count,
-                                            rmm::device_buffer&& null_mask,
-                                            rmm::cuda_stream_view stream,
-                                            rmm::mr::device_memory_resource* mr)
```

Comment:
surprised! how was this missed during 'correct unused parameter warnings' PRs?

Reply:
I'm wondering that too. I think the nvcc compiler is not as sophisticated as the gcc compiler yet perhaps.

Reply:
The builtin nvcc detection for unused parameters only occurs on 'internal' functions, the function isn't internal ( for example not in an anonymous namespaces ) so nvcc doesn't warn about it.

Reply:
So is there any compiler flag to turn that up?